oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/json/JsopBuilder.java

```diff
@@ -28,7 +28,7 @@ public class JsopBuilder implements JsopWriter {
 
     private static final boolean JSON_NEWLINES = false;
 
-    private StringBuilder buff = new StringBuilder();
+    private final StringBuilder buff = new StringBuilder();
```

Comment:
what is this change about?

Reply:
The field is not changed, so it should be final to convey the intention that it should not change and to get the compiler to enforce this invariant. This was causing a warning in IntelliJ and as it was in the same file that I was modifying, I added this small improvement.

Reply:
Please: no. Changes for a ticket should only for the issue desribed in the ticket (or extend the ticket description).

Reply:
reverted.